docs/configuration.rst

```diff
@@ -42,7 +42,14 @@ You can now add it to your path with::
 Windows
 ^^^^^^^
 
-.. todo:: If you know how to do this on windows, pull request welcome I got nothing.
+After downloading the binary on Windows, you can immediately run it.
+
+We recommend that you rename the executable to ``rack``, copy it to a location outside of your Downloads folder (e.g. C:\\tools) and add that location to your PATH. You must open a new command prompt after modifying the PATH variable.
```

Comment:
Do you mean `rack.exe` here or is it ok for us to create binaries without the suffix for Windows?